scripts/ceph-windows/setup_libvirt_ubuntu_vm

```diff
@@ -32,6 +32,7 @@ ssh_authorized_keys:
 
 packages_update: true
 packages:
+  - libvirt-daemon-system
```

Comment:
> The issue is that we're assuming Libvirt to be preinstalled, which isn't necessarily the case.

We are always downloading the same image, from https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img, right?  Did it get updated recently?

Reply:
My bad, this shouldn't go there. It's actually supposed to be installed on the host, updating right now.

Reply:
I think it's just that some Jenkins slaves happened to have it preinstalled, so the scripts assumed libvirt to be already available.